lib/shared/drive-constraints.js

```diff
@@ -142,7 +142,31 @@ exports.isSourceDrive = (drive, image) => {
  * }
  */
 exports.isDriveLargeEnough = (drive, image) => {
-  return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'size' ], UNKNOWN_SIZE);
+  const driveSize = _.get(drive, [ 'size' ], UNKNOWN_SIZE);
+
+  if (_.get(image, [ 'size', 'final', 'estimation' ])) {
+
+    // If the original image size is larger than the drive size, and
```

Comment:
Another minor nit, but it _might_ be worth updating the comment to match the code? i.e. "If the drive size is smaller than the original image size, ..."